frontend/src/util/attribution.ts

```diff
@@ -1,7 +1,6 @@
+import { GenerateSecureID } from '@highlight-run/client/src/utils/secure-id'
 import Cookies from 'js-cookie'
 
-import { GenerateSecureID } from '../../../sdk/client/src/utils/secure-id'
```

Comment:
Changed this because relative imports outside the package root is incompatible with Reflame's stricter package isolation, and as far as I could see, there's nothing that necessitates their use here.